lib/Makefile

```diff
@@ -254,7 +254,7 @@ oxli.pc: oxli.pc.in
 	sed -e 's,@prefix@,$(PREFIX),'  -e 's,@VERSION@,$(VERSION),' $< >$@
 
 $(ZLIB_OBJS):
-	(cd $(ZLIB_DIR) && ./configure && make $(ZLIB_OBJS_BASE))
+	(cd $(ZLIB_DIR) && ./configure && make -f Makefile.pic $(ZLIB_OBJS_BASE))
```

Comment:
why this? (no objections, just rather leary of stuff that changes semantics along with syntax in the same large PR, at this point.) Is this going to break a bunch of installs?